### PR TITLE
Fix external images flaws in Web/CSS

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

**Comment by peterbe** [comment]
Note, there are still some flaws in these 10 pages but they're at least not external images.

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/css/_doublecolon_-moz-range-progress/index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@ <h3 id="Result">Result</h3>
 
 <p>A progress bar using this style might look something like this:</p>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/12041/Screen%20Shot%202015-12-04%20at%2020.14.48.png"></p>
+<p><img alt="" src="screen_shot_2015-12-04_at_20.14.48.png"></p>
```
Comment (ericwbailey) [inline files/en-us/web/css/_doublecolon_-moz-range-progress/index.html:45]: Can we provide an alt description for this?

File: files/en-us/web/css/_doublecolon_-moz-range-thumb/index.html [hunk 2]
```diff
@@ -41,7 +41,7 @@ <h3 id="Result">Result</h3>
 
 <p>A progress bar using this style might look something like this:</p>
 
-<p><img alt="The thumb of the &lt;input type=right> styled in green" src="https://mdn.mozillademos.org/files/12027/Screen%20Shot%202015-12-04%20at%2013.30.08.png"></p>
+<p><img alt="The thumb of the &lt;input type=right> styled in green" src="screen_shot_2015-12-04_at_13.30.08.png"></p>
```
Comment (ericwbailey) [inline files/en-us/web/css/_doublecolon_-moz-range-thumb/index.html:44]: I weirdly don't have the ability to make a suggestion here. Can we update the closing `>` in the alt description, and add a period to the end?

File: files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 <p class="summary">Breadcrumb navigation helps the user to understand their location in the website by providing a breadcrumb trail back to the start page.</p>
 
-<p><img alt="Links displayed inline with separators" src="https://mdn.mozillademos.org/files/16228/breadcrumb-navigation.png" style="height: 108px; width: 1268px;"></p>
+<p><img alt="Links displayed inline with separators" src="breadcrumb-navigation.png"></p>
```
Comment (ericwbailey) [inline files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.html:16]: Can we add a period to the end of this alt description?

File: files/en-us/web/css/layout_cookbook/media_objects/index.html [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
 <p class="summary">The <em>Media Object</em> is a pattern we see all over the web. <a href="http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/">Named by Nicole Sullivan</a> it refers to a two-column box with an image on one side and descriptive text on the other, e.g. a facebook post or tweet.</p>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/16112/media-object.png"></p>
+<p><img alt="" src="media-object.png"></p>
```
Comment (ericwbailey) [inline files/en-us/web/css/layout_cookbook/media_objects/index.html:18]: Can we add an alt description?

File: files/en-us/web/css/layout_cookbook/pagination/index.html [hunk 5]
```diff
@@ -13,7 +13,7 @@
 
 <p class="summary">This cookbook pattern demonstrates the navigation pattern used to display pagination, where the user can move between pages of content such as search results.</p>
 
-<p><img alt="Links to sets of pages in a paged listing" src="https://mdn.mozillademos.org/files/16279/pagination.png" style="height: 192px; width: 950px;"></p>
+<p><img alt="Links to sets of pages in a paged listing" src="pagination.png"></p>
```
Comment (ericwbailey) [inline files/en-us/web/css/layout_cookbook/pagination/index.html:16]: Can we add a period to the end of this alt description?

File: files/en-us/web/css/mask-border/index.html [hunk 6]
```diff
@@ -73,7 +73,7 @@ <h3 id="Setting_a_bitmap-based_mask_border">Setting a bitmap-based mask border</
 
 <p>In this example, we will mask an element's border with a diamond pattern. The source for the mask is a ".png" file of 90 by 90 pixels, with three diamonds going vertically and horizontally:</p>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/15836/mask-border-diamonds.png" style="height: 90px; width: 90px;"></p>
+<p><img alt="" src="mask-border-diamonds.png"></p>
```
Comment (ericwbailey) [inline files/en-us/web/css/mask-border/index.html:76]: Can we provide an alt description for this?

File: files/en-us/web/css/position_value/index.html [hunk 7]
```diff
@@ -19,7 +19,7 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/12215/position_type.png" style="float: left; height: 230px; width: 362px;">The <code>&lt;position&gt;</code> data type is specified with one or two keywords, with optional offsets.</p>
+<p><img alt="" src="position_type.png" style="float: left;">The <code>&lt;position&gt;</code> data type is specified with one or two keywords, with optional offsets.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/position_value/index.html:22]: Can we provide an alt description for this?

**Review by ericwbailey — COMMENTED** [review]
This is looking good to me. Some feedback on alt descriptions that I'd like to see updated as long as work is being done in these files.

**Comment by ericwbailey** [comment]
Shoot, didn't see this was merged already. Ignore me!

**Comment by Ryuno-Ki** [comment]
@ericwbailey Alternatively, create a follow-up PR :)

**Comment by peterbe** [comment]
@ericwbailey Sorry to hear that you missed the fact that it got merged. That's annoying. 

I actually only ran the fixable flaws on these pages. So the `alt=""` was unfortunately already present and still is :(

**Pull request merged**